frontend/package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "build": "next build",
     "dev": "next dev",
+    "debug": "NODE_OPTIONS='--inspect' next dev",
```

Comment:
I was not able to get this to work to use the chrome node debugger, and ended up using the VS code debugger, which did work to debug node based / server side next functions. If anyone else can get this to work, we should probably keep this, or we can just get rid of it.

see https://github.com/vercel/next.js/issues/48767 and https://nextjs.org/docs/pages/building-your-application/configuring/debugging